pkg/cmd/personal-token/delete/detele.go

```diff
@@ -0,0 +1,51 @@
+package delete
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	msg "github.com/aziontech/azion-cli/messages/personal-token"
+	api "github.com/aziontech/azion-cli/pkg/api/personal_token"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	var id string
+
+	deleteCmd := &cobra.Command{
+		Use:           msg.DeleteUsage,
+		Short:         msg.DeleteShortDescription,
+		Long:          msg.DeleteLongDescription,
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		Example: heredoc.Doc(`
+		$ azioncli personal_token delete --id 7a187044-4a00-4a4a-93ed-d230900421f3
+		$ azioncli personal_token delete -i 7a187044-4a00-4a4a-93ed-d230900421f3
```

Comment:
```suggestion
		$ azion personal_token delete -i 7a187044-4a00-4a4a-93ed-d230900421f3
```

Reply:
I have a question. Wouldn't the shortened flags be used only for global vars?

Reply:
As far as I know, shorthand are for IDs too

Reply:
It's worth checking with Tiago, if that idea will continue in azion 1.0